PR #21 Reformatted (#128)
mongodb/docs-meta: source/style-guide/markup/headings.txt

=====================
Headings and Sections
=====================

Headings organize information into a hierarchy of sections.
To set headings, add repeating characters below the heading text.

> Learn more about headings from the
> `Sphinx documentation <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections>`__.

Reviewer: ```suggestion
To learn more about headings, see the
`Sphinx documentation <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections>`__.
```

Heading Markup Rules
--------------------

To avoid errors in your header markup syntax, stick to the following rules:

- Don't skip heading levels. Heading level 3 can't come after heading level 1.

  .. list-table::
     :header-rows: 1
     :widths: 50 50

     * - Use
       - Avoid

     * -

         .. code-block:: rst

            ============
            Manage Users
            ============

            Create One New User
            -------------------

            Options
            ~~~~~~~
       -

         .. code-block:: rst
            :copyable: false

            ============
            Manage Users
            ============

            Create One New User
            ~~~~~~~~~~~~~~~~~~~

- Make the underline as long as the text above it. If you use a source constant

>   in the header, you need the line to match the length of the resolved text.

Reviewer: **Style:** See [Write Clear and Concise Sentences](https://www.mongodb.com/docs/meta/style-guide/writing/write-clear-concise-sentences-paragraphs/)

```suggestion
  in the header, the line must match the length of the resolved text.
```
